MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -773,6 +773,35 @@ extension MUXSDKPlayerBinding {
         self.dispatcher.dispatchEvent(event, forPlayer: self.name)
         self.state = .viewEnd
     }
+    
+    func dispatchOrientationChange(orientation: MUXSDKViewOrientation) {
+        guard let player = self.player else {
+            print("MUXSDK-ERROR - Mux failed to find the Kaltura Playkit Player for player name: \(self.name)")
+            return
+        }
+        
+        self.updateVideoData(player: player)
+        
+        var orientationData: MUXSDKViewDeviceOrientationData? = nil
+        
+        switch orientation {
+        case.landscape:
+            orientationData = MUXSDKViewDeviceOrientationData(z: 0.0)
+        case .portrait:
+            orientationData = MUXSDKViewDeviceOrientationData(z: 90.0)
+        case .unknown:
```

Comment:
let's log a warning here also?

Reply:
I think you're right we should not support unknown. Im removing it.